scripts/release_dev.js

```diff
@@ -1,62 +1,55 @@
+const { execSync } = require('child_process');
 const path = require('path');
-const { askForConfirmation, askForText, runExec } = require('./release.helpers');
 
-const packageJsonPath = path.resolve(__dirname, '../package.json');
-const packageJson = require(packageJsonPath);
+const semver = require('semver');
 
-const versionType = process.argv.slice(2)[0];
-const isVersionType = (type) => versionType === type;
+const {
+  askForConfirmation,
+  askForText,
+  checkGitStatus,
+  checkNpmAuth,
+  revertCommit,
+  runExec,
+} = require('./release.helpers');
 
-function getDateYyyyMMDDHHMMSS() {
-  function pad2(n) {
-    // always returns a string
-    return (n < 10 ? '0' : '') + n;
-  }
-  const date = new Date(); // MAKE IT UTC!
-  return (
-    date.getFullYear() +
-    pad2(date.getMonth() + 1) +
-    pad2(date.getDate()) +
-    '-' +
-    pad2(date.getHours()) +
-    pad2(date.getMinutes()) +
-    pad2(date.getSeconds())
-  );
-}
+const packageJsonPath = path.resolve(__dirname, '../package.json');
+const packageJson = require(packageJsonPath);
 
-const currentDate = getDateYyyyMMDDHHMMSS();
+async function getNewVersion() {
+  const result = await execSync('git rev-parse --short HEAD^');
```

Comment:
do you still need the await if its `Sync`?

Reply:
No, this was old code, it's gone